src/credential/Credential.spec.ts

```diff
@@ -37,17 +37,19 @@ describe('Credential', () => {
 
     // cannot be used since the variable needs to be established in the outer scope
     // eslint-disable-next-line prefer-destructuring
-    reqForAtt = (await RequestForAttestation.fromClaimAndIdentity(
-      Claim.fromCTypeAndClaimContents(
-        ctype,
-        {
-          name: 'Peter',
-          age: 12,
-        },
-        claimer.getAddress()
-      ),
-      claimer
-    )).message
+    reqForAtt = (
+      await RequestForAttestation.fromClaimAndIdentity(
+        Claim.fromCTypeAndClaimContents(
+          ctype,
+          {
+            name: 'Peter',
+            age: 12,
+          },
+          claimer.getAddress()
+        ),
+        claimer
+      )
+    ).message
```

Comment:
Need to use that newly gained knowledge! 

```suggestion
    ;({ message: reqForAtt } = (
      await RequestForAttestation.fromClaimAndIdentity(
        Claim.fromCTypeAndClaimContents(
          ctype,
          {
            name: 'Peter',
            age: 12,
          },
          claimer.getAddress()
        ),
        claimer
      )
    ))
```

Reply:
Fixed in 4e9832e